go/common/host/services.go

```diff
@@ -157,3 +149,16 @@ type LogSubscriptionManager interface {
 	Unsubscribe(id rpc.ID)
 	SendLogsToSubscribers(result *common.EncryptedSubscriptionLogs)
 }
+
+// Metrics service doesn't currently expose any methods internally to the host, its role is just to serve metrics data via HTTP
+type Metrics interface {
+	Registry() metrics.Registry
+}
+
+// RPCServer doesn't currently expose any methods internally to the host, its role is just to serve requests
+type RPCServer interface{}
```

Comment:
Is this meant to be and interface{} ?

Reply:
I was keen to give it a type but the RPCServer service doesn't expose any methods to the rest of the host currently